opal/util/misc.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024      Amazon.com, Inc. or its affiliates.
+ *                         All Rights reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+/** @file:
+ * Miscellaneous utilities
+ */
+
+#ifndef OPAL_UTIL_MISC_H
+#define OPAL_UTIL_MISC_H
+
+#include "opal/include/opal_config.h"
```

Comment:
Headers don't include opal_config.h

Reply:
Good catch. My editor misleadingly drew a squiggle under on `BEGIN_C_DECLS` when I didn't include opal_config.h.

Removed in update. Also need to audit other headers on main and v5.0.x.